src/main.cpp

```diff
@@ -1028,11 +1028,8 @@ static void handleAddressAfterInit(CNode *pfrom)
         }
 
         // Get recent addresses
-        if (pfrom->fOneShot || pfrom->nVersion >= CADDR_TIME_VERSION || addrman.size() < 1000)
-        {
```

Comment:
Shoudn't we still need to check for `pfrom->fOneShot` or `addrman.size() < 1000` ?

Reply:
since pfrom->nVersion was is always higher than CADDR_TIME_VERSION this always evaluated to true so i just removed it.

Reply:
you are right indeed